apps/docs/spec/cli_v1_commands.yaml

```diff
@@ -2466,42 +2491,9 @@ commands:
     summary: Generate types from Postgres schema
     tags: []
     links: []
-    subcommands:
-      - supabase-gen-types-typescript
-    flags: []
-  - id: supabase-gen-types-typescript
-    title: supabase gen types typescript
-    summary: Generate types for TypeScript
-    tags: []
-    links: []
-    usage: supabase gen types typescript [flags]
+    usage: supabase gen types [flags]
     subcommands: []
-    flags:
-      - id: db-url
```

Comment:
@sweatybridge are all these flags meant to have disappeared?

Reply:
nope, it's unintentional. I will address it and update this pr